index/index.go

```diff
@@ -126,6 +126,9 @@ func (i *Index) Query(ctx context.Context, query string, opts ...option.Option)
 	if err != nil {
 		return nil, err
 	}
+	if len(embeddings) == 0 {
```

Comment:
What embedder did you use? It looks like your embedded returned no embeddings an no error here.